decidim-core/lib/decidim/core.rb

```diff
@@ -270,9 +270,21 @@ def self.seed!
     false
   end
 
-  # How long can a user remained logged in before the session expires
+  # How long can a user remained logged in before the session expires. Notice that
+  # this is also maximum time that user can idle before getting automatically signed out.
   config_accessor :expire_session_after do
-    1.day
+    30.minutes
```

Comment:
```suggestion
    24.hours
```

Reply:
I would suggest keeping the default at 30 minutes and letting the admin to raise the limit if necessary.

Decidim is used also on public computers (e.g. libraries) where people don't always remember to log out. Having this setting at 24 hours makes it easier to hijack user profiles on public computers.

Reply:
After we've tried this in multiple staging servers we find out that 30.minutes by default is too short time for working with Decidim.

Can you change it to 24 hours/1 day 🙏🏽 ?

Reply:
@andreslucena How about if something happens on the screen (e.g. user moves the mouse, scrolls the view), we extend the period after specific intervals?

So, every 5 minutes check if the user has moved the mouse or scrolled the view in the past 5 minutes. If so, extend the session timeout with 5 minutes (we send a heartbeat request which causes the session to extend to 30 minutes).

What we are trying to do here is to solve a potential security issue where the user leaves their session available for hijacking e.g. on public computers.

Reply:
Ping @andreslucena, could you answer to the questions above?

Reply:
@andreslucena Could you please comment the above?

Reply:
@andreslucena Could you also chime in here? This has been pending for so long.

Reply:
Sorry for my delay, I'll review my filters so I get this kind of pings 😞 

> @andreslucena How about if something happens on the screen (e.g. user moves the mouse, scrolls the view), we extend the period after specific intervals?

That'd be great. My main fear with these was in pages with too much text and by scrolling/interacting somehow then that'd be mitigated. 👍🏽